cfgov/mega_menu/jinja2tags.py

```diff
@@ -0,0 +1,53 @@
+from django.template import loader
+from django.utils.translation import get_language_from_request
+
+from flags.state import flag_enabled
+from jinja2 import Markup, contextfunction
+from jinja2.ext import Extension
+
+from mega_menu.frontend_conversion import FrontendConverter
+from mega_menu.models import Menu
+
+
+def mega_menu_template(context):
+    request = context.get('request')
+
+    if flag_enabled('MEGA_MENU_VAR_1', request=request):
+        return '_includes/organisms/mega-menu-var-1.html'
+    elif flag_enabled('MEGA_MENU_VAR_2', request=request):
+        return '_includes/organisms/mega-menu-var-2.html'
+    else:
+        return '_includes/organisms/mega-menu.html'
+
+
+def select_menu_for_context(context):
+    language = context.get('language')
+
+    if not language:
+        request = context['request']
+        language = get_language_from_request(request, check_path=True)
+
+    return Menu.objects.get(language=language)
```

Comment:
This line generates an error for all of the other language pages linked in the top bar of the site (Chinese, Vietnamese, Korean, etc): `mega_menu.models.DoesNotExist: Menu matching query does not exist.`

Reply:
Good catch, thanks. I was being too clever with this. I didn't properly handle pages in other languages besides English and Spanish. I've pushed a proposed change in ad0173161; to test, visit e.g http://localhost:8000/language/vi/.